apps/aot/src/components/CountdownTimer.tsx

```diff
@@ -21,6 +21,10 @@ const useAotCountdown = () => {
     }
 
     const countdown = () => {
+      if (window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
+        return;
+      }
```

Comment:
For this and the other `prefers-reduced-motion` changes: https://web.dev/articles/prefers-reduced-motion.

It would be better to add a play/pause button for animations: https://www.w3.org/WAI/tutorials/carousels/animations. That way people (like me!) who have a hard time using a using a webpage when there are distracting animations don't have to opt out of them altogether.

Reply:
ohh TIL